src/modules/storage/Storages.ts

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+
+    The superclass of storages.
+
+    This has a DB instance.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as sqlite from 'sqlite3';
+
+export class Storages
+{
+    /**
+     *  The instance of sqlite3
+     */
+    protected db: sqlite.Database;
+
+    /**
+     * Constructor
+     * @param filename Valid values are filenames,
+     * ":memory:" for an anonymous in-memory database and
+     * an empty string for an anonymous disk-based database
+     * @param callback : If provided, this function will be called when
+     * the database was opened successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    constructor (filename : string = ".cache\ledger", callback : any = null)
```

Comment:
filename should not be a default parameter if you want this class to be reusable. What if you forget to set the filename? then you end up overwriting the ledger storage by mistake.

Reply:
Thank you reviews.
^ Fixed the DB filename.